build/gen_stub.php

```diff
@@ -737,6 +743,7 @@ function generateCodeWithConditions(
 }
 
 function generateArgInfoCode(FileInfo $fileInfo): string {
+    $generatedDeclarations = [];
```

Comment:
This variable can be moved closer to where it is used.

Reply:
Forgot to do this - will fix soon